include.js

```diff
@@ -1814,6 +1814,13 @@ let cards = [
     imageLink: './Art/Sayed-Husain/hover.gif',
     author: 'Sayed-Husain',
     githubLink: 'https://github.com/Sayed-Husain'
+  },
+  {
+    artName: 'Click Animation',
+    pageLink: './Art/JohnRuben717/index.html',
+    imageLink: './Art/Sayed-JohnRuben717/Click-Animation.gif',
```

Comment:
This seems to contain a typo/error. The folder name is `JohnRuben717`, so the image link should point to that:
```js
imageLink: './Art/JohnRuben717/Click-Animation.gif',
```